src/browser/Navigators/Navigator/WebView/MozBrowserFrame.js

```diff
@@ -185,7 +185,7 @@ const decodeLocationChange =
     , canGoForward: null
     }
   : { type: "LocationChanged"
-    , uri: detail.uri
+    , url: detail.url
```

Comment:
You don't want to change the decoded property name.

So more like: `uri: detail.url || detail.uri || detail` (for backward compatibility with previous version of Servo and with Gecko).